input/RadiativeTransfer/iliev3.in

```diff
@@ -0,0 +1,294 @@
+         value = [ 7.719575430051935e+62,  x  <=  0.000000000000000 , 1.308402615263040e+42 ];
+     };
+     N0 {
+         field_list = "photon_density_0";
+         type = "inflow";
+         value = [ 1.259813889082893e+63,  x  <=  0.000000000000000 , 8.449277459299774e+50 ];
+     };
+     N1 {
+         field_list = "photon_density_1";
+         type = "inflow";
+         value = [ 1.392277541849998e+63,  x  <=  0.000000000000000 , 8.449277459299774e+50 ];
+     };
+     N2 {
+         field_list = "photon_density_2";
+         type = "inflow";
+         value = [ 1.662841598565787e+62,  x  <=  0.000000000000000 , 8.449277459299774e+50 ];
+     };
+     list = [ "N0", "N1", "N2", "Fx0", "Fx1", "Fx2" ];
+     x_upper {
+         axis = "x";
+         face = "upper";
+         type = "outflow";
+     };
+     y_periodic {
+         axis = "y";
+         type = "periodic";
+     };
+     z_periodic {
+         axis = "z";
+         type = "periodic";
+     };
+ }
+
+ Domain {
+     lower = [ 0.000000000000000, 0.000000000000000, 0.000000000000000 ];
+     upper = [ 1.000000000000000, 1.000000000000000, 1.000000000000000 ];
+ }
+
+ Field {
+     alignment = 8;
+     gamma = 1.400000000000000;
+     ghost_depth = 4;
+     list = [ "density", "internal_energy", "total_energy", "velocity_x", "velocity_y", "velocity_z", "pressure", "temperature", "photon_density", "flux_x", "flux_y", "flux_z", "photon_density_0", "flux_x_0", "flux_y_0", "flux_z_0", "photon_density_1", "flux_x_1", "flux_y_1", "flux_z_1", "photon_density_2", "flux_x_2", "flux_y_2", "flux_z_2", "metal_density", "HI_density", "HII_density", "HeI_density", "HeII_density", "HeIII_density", "e_density", "RT_heating_rate", "RT_HI_ionization_rate", "RT_HeI_ionization_rate", "RT_HeII_ionization_rate" ];
+ }
+
+ Group {
+     color {
+         field_list = [ "HI_density", "HII_density", "HeI_density", "HeII_density", "HeIII_density", "e_density" ];
+     };
+     derived {
+         field_list = [ "temperature", "pressure" ];
+     };
+     list = [ "color", "derived" ];
+ }
+
+ Initial {
+     list = [ "value", "m1_closure" ];
+     value {
+         HII_density = 0.000000000000000;
+         HI_density = [ 0.04000000000000000,  sqrt(((( x  -  0.7575757575600000 ) * ( x  -  0.7575757575760000 )) + (( y  -  0.5000000000000000 ) * ( y  -  0.5000000000000000 ))) + (( z  -  0.5000000000000000 ) * ( z  -  0.5000000000000000 )))  <=  0.1212121212121000 , 0.0002000000000000000 ];
+         HeIII_density = 0.000000000000000;
+         HeII_density = 0.000000000000000;
+         HeI_density = 0.000000000000000;
+         density = [ 0.04000000000000000,  sqrt(((( x  -  0.7575757575600000 ) * ( x  -  0.7575757575760000 )) + (( y  -  0.5000000000000000 ) * ( y  -  0.5000000000000000 ))) + (( z  -  0.5000000000000000 ) * ( z  -  0.5000000000000000 )))  <=  0.1212121212121000 , 0.0002000000000000000 ];
+         e_density = 0.000000000000000;
+         flux_x_0 = 1.308402615263040e+42;
```

Comment:
I got these numbers for photon densities and fluxes by converting a tiny number in physical units to code units (1e-16 cm^-3 = 8.4e50 code_length^-3), which are still tiny when compared to the values on the left-x boundary. Would probably be a lot more clear to make these obviously tiny numbers. 

The values currently have to be nonzero though, otherwise you get NaNs in `EnzoMethodM1Closure::get_reduced_variables()`